tests/test_self.py

```diff
@@ -724,10 +724,34 @@ def test_sys_path() -> Generator[None, None, None]:
             finally:
                 sys.path = original_path
 
+        @contextlib.contextmanager
+        def test_environ_pythonpath(
+            new_pythonpath: Optional[str],
+        ) -> Generator[None, None, None]:
+            if new_pythonpath:
+                os.environ["PYTHONPATH"] = new_pythonpath
+            try:
+                yield
+            finally:
+                if new_pythonpath:
+                    del os.environ["PYTHONPATH"]
```

Comment:
Doesn't that break `PYTHONPATH` for subsequent tests?

Reply:
It might, good point. Although I guess with Github actions `PYTHONPATH` is already `None` so it doesn't matter

Reply:
Ahh. This might be the issue why issue why the tests are failing for you.

Reply:
Yes, I thought so :)